src/Orleans.Runtime/GrainDirectory/LocalGrainDirectory.cs

```diff
@@ -569,26 +613,17 @@ public Task UnregisterAfterNonexistingActivation(ActivationAddress addr, SiloAdd
 
         public async Task UnregisterAsync(ActivationAddress address, UnregistrationCause cause, int hopCount)
         {
+            var callerMembershipVersion = default(MembershipVersion);
```

Comment:
The idea is that eventually we will send the caller's membership version along with requests, but for now we declare it to be 'default', which indicates to `GetPartitionOwner` that if the local silo does not believe it's the correct silo for a request then it will refresh to the latest membership before continuing.

Reply:
Should this be turned to a comment in code? I mean it's easier to read comments in code rather than Git history.

Reply:
Possibly the first part (~"eventually pass this as a parameter") should, but the second part is an implementation detail of GetPartitonOwner